catalog/app/containers/Bucket/requests/packageUpload.ts

```diff
@@ -0,0 +1,258 @@
+import * as R from 'ramda'
```

Comment:
i think we need to use some more generic name for this file, bc its contents doesnt relate only to uploading, maybe smth like `packageManipulation` or `packageCreation`

Reply:
Maybe, `requests/package.ts` then?

Reply:
sounds good